src/atoms/InputLabel/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as Label } from "./InputLabel";
```

Comment:
Modules should export a function with the same name so this should be `as InputLabel` for consistency or we can change all of the file names just to Label but that seems like a pain